extra/integration-testing/test-compat/docker-compose.compat-2.0.yml

```diff
@@ -0,0 +1,17 @@
+version: '2.1'
```

Comment:
What about the files we already have? Shouldn't these be moved, rather than copied?

Reply:
Oh sorry, I was viewing with `-C` and git tricked me. Strike that.

New question instead: Do we actually need the template? Looks to me like we can just remove it.

Reply:
no worries. thought to preserve for reference. removed.

Reply:
The same content is in the new files you created, so I think removing it is fine! 👍